_pages/dev/styling-and-page-layout/skeleton-design.md

```diff
@@ -0,0 +1,79 @@
+
+Spartacus makes use of skeleton design only in a limited number of spaces. The skeleton design feature is fully compatible with other UX patterns, such as responsive design, theming, directionality, and so on.
+
+Skeleton design might make traditional progress bars obsolete, although the two can coexist. Spartacus has not changed the progress bar loader, and there is no plan to remove it.
+
+The following are some of the requirements that were taken into account while this feature was being developed:
+
+- Skeleton design should work on top of existing UX patterns, such as responsive design, theming, directionality, and so on.
+- Multiple skeletons should be able to run in parallel.
+- The animation should often include multiple elements, which means that a single animation goes from the left to the right of the outer element.
+- If multiple skeletons are on the same page, it should be possible to have multiple skeleton animations.
+- The animation should always go from left to right, even on a right-to-left oriented site.
+
+## Rendering Skeleton Designs
+
+There are various techniques that you can use to render skeleton designs, such as the following:
+
+- using an image (svg) to render a skeleton while loading the content
+- using an special skeleton component to render a skeleton for a specific component
+- using CSS to render a skeleton for an existing component
+
+Introducing specific skeleton images or components has various disadvantages. They can cause problems for the implementation and the performance, and they do not cope with other UX patterns. However, using a pure CSS solution on an existing component will work with all existing UX patterns, and will not add more complexity or performance drawbacks. This is done with the following techniques:
+
+- A `ghost` CSS class is added to the root element of the specific skeleton. If you use multiple isolated skeletons on a single page, those can work independently. Each `ghost` has its own skeleton animation.
+- The existing component DOM is used for the skeleton elements. No special DOM or specific ghost elements are introduced to build the skeletons.
+- The content stream (or streams) must be started with a ghost data set. Data streams in Spartacus are based on RxJS, which uses a `startWith` operator. The `startWith` operator typically has an array of empty objects, and potential other elements, to mimic the data.
+- The component implementation should be prepared for empty elements. This is a good practice anyway, but might not be the case for existing components.
+
+## Example
+
+An example of skeleton design can be found in the organization feature. Most entities in the organization self-service are based on a list of items that can be managed by the user. These lists use two techniques to implement skeletons, as described in the following sections. Standard CSS is used to add the gray boxes with an animation.
```

Comment:
> These lists use two techniques to implement skeletons...

We must change this a bit:

The implementation of these lists leverage the skeleton design feature of Spartacus. There are three techniques being used:
- using a ghost data set to mimic the list data 
- adding the `ghost` css class to apply default skeleton styles
- adding bespoke css rules for specifics to the organization list skeleton layout

The first two techniques are further detailed below.